Alpaca.Markets/DataStreamingClientBase.cs

```diff
@@ -31,7 +31,7 @@ public IEnumerable<String> Streams
             get { yield return _stream; }
         }
 
-        public Boolean Subscribed { get; private set; }
+        public long? Subscribed { get; private set; }
```

Comment:
Breaking change! Never do so for the public API - what should do other SDK users who already have code that uses this property? I'll never accept any PR with breaking changes. If you want to add the new property - it's OK but changing the existing one without any serious reason is not a good idea.